src/configure.js

```diff
@@ -1,4 +1,4 @@
-function configure(apiKey) {
+export default function configure(apiKey) {
```

Comment:
As far as I can see, the `configure` function is now dead code and can be removed entirely.